src/tools/xml.cpp

```diff
@@ -54,7 +54,7 @@ xmlWriter_c::xmlWriter_c(std::ostream & str) : stream(str), state(StBase)
   stream << "<?xml version=\"1.0\"?>\n";
 }
 
-xmlWriter_c::~xmlWriter_c(void) noexcept(false)
```

Comment:
This change seems to be for an ancient C++ version; I would like to keep the code clean wrt at least c++11.
```
tools/xml.cpp: In destructor ‘xmlWriter_c::~xmlWriter_c()’:
tools/xml.cpp:60:5: warning: ‘throw’ will always call ‘terminate’ [-Wterminate]
   60 |     throw xmlWriterException_c("Not all tags were closed");
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
tools/xml.cpp:60:5: note: in C++11 destructors default to ‘noexcept’
```